src/main/java/digital/slovensko/autogram/ui/gui/GUI.java

```diff
@@ -256,4 +257,17 @@ public void focusJob(SigningJob job) {
     private Window getJobWindow(SigningJob job) {
         return jobControllers.get(job).mainBox.getScene().getWindow();
     }
+
+    public void sign(File file, Autogram autogram) {
```

Comment:
podobne ako ostatne pripomienky ku ui.sign() myslim ze by sa dala urobit metoda `autogram.sign(File file)` alebo aspon namiesto `this.sign(autogram, job)` pouzit `autogram.sign(job)`